src/app/frontend/replicationcontrollerdetail/serviceendpoint_directive.js

```diff
@@ -22,6 +22,7 @@ export default function serviceEndpointDirective() {
     templateUrl: 'replicationcontrollerdetail/serviceendpoint.html',
     scope: {
       'endpoint': '=',
+      'url': '@',
```

Comment:
I was thinking about some heuristic that would actually establish if `endpoint` has something that is worth showing as a link. Checking common ports (HTTP/HTTPS) and maybe trying to actually check if this link points to something that browser can display.

Checking this as true in code would make link available for everything and this is not desired I guess.

What do you think?